binding/web/src/types.ts

```diff
@@ -40,7 +36,7 @@ export type LeopardWorkerInitRequest = {
   command: 'init';
   accessKey: string;
   modelPath: string;
-  initConfig: LeopardInitConfig;
+  options: LeopardOptions;
```

Comment:
I also noticed that not using the same name for `LeopardTranscription`. In web and Node it's `LeopardTranscription`, but in JNI it was `LeopardTranscript`. Can we make it `LeopardTranscript` instead?